### DISCUSSION TITLE
Do you have  jssdk?

### DISCUSSION THREAD

**Discussion by MorningStarHu** [epam/ketcher#1479] [General]
I want to use it in a project other than React. What should I do?

**Comment by AndreiMazol** [comment]
Dear @MorningStarHu ,

Ketcher team is now working on ketcher-core package that  allows to integrate with any FE framework/library (not only react presentation layer).
It should be done in the nearest releases, all required documentation will be presented on Wiki pages.

Regards,
Ketcher team

**Comment by MorningStarHu** [comment]
> 

thanks，and now i want use ketcher in . HTML file ,Can it be achieved now？I haven't seen the document for a long time

**Comment by AndreiMazol** [comment]
Unfortunately, right now Ketcher is available only  as React Component Library. To integrate it into HTML without reactjs dependency, I recommend you use IFrame and ready-to-run app (see assets https://github.com/epam/ketcher/releases/tag/v2.4.2).  Please look at  examples here:
 https://lifescience.opensource.epam.com/ketcher/demo.html

**Comment by AndreiMazol** [comment]
Please look at instruction here to build Ketcher locally: https://github.com/epam/ketcher/blob/master/DEVNOTES.md

If you want to  run ready-to-run app follow the next steps:
- download required artifacts( for example ketcher-standalone-2.4.2.zip from https://github.com/epam/ketcher/releases/tag/v2.4.2)
- unzip it to `\ketcher-standalone-2.4.2`
- run `npx serve` from `\ketcher-standalone-2.4.2\standalone`
- open generated link in browser
